docs/howto/ios-tips.md

```diff
@@ -42,6 +46,16 @@ obtain Apple Developer credentials that will allow you to sign the app.
 Register at https://developer.apple.com. Then use your Apple ID in Xcode
 and choose it as your `Signing > Team` for both ZulipMobile and ZulipMobileTests.
 
+If it's not your first time, and you haven't changed anything in the
+`ios` folder yourself, you might see build failures caused by
+interfering residue from a previous build. If you do, clear Xcode's
+build folder for ZulipMobile with `rm -rf`, or through Finder. The
+default path for the build folder is
+`~/Library/Developer/Xcode/DerivedData/ZulipMobile-diuocloqwafvdpeozujwphdrhalf`
+(the hash at the end will be different), but you can find it in Xcode
+at File > Workspace Settings > Build Location. If that doesn't fix it,
+see our [build failure troubleshooting doc][build-run-troubleshoot].
+
```

Comment:
It'd be good to avoid duplicating these details -- let's find a good home for them, and then from elsewhere we can link to that (with just enough words around each link to make it easy to find when it's what you need).

Probably making a section in this `ios-tips` doc, like `### Clearing the build folder`, would be a good place.

Reply:
> (with just enough words around each link to make it easy to find when it's what you need)

Sure, makes sense. I agree that its home should be in `ios-tips`, but it definitely needs a shoutout from Troubleshooting, as that's where people go for all other build failures.